### PR TITLE
Feat/reviewed at filter

### PR DESCRIPTION
**Pull request by andreLumor**
Resolves: #130 

What was done:
Fixed contributions seeds that broke on #124 (now seeded contributions as self reviewed)
Removed state filter
Removed Company from table

Updated Filters:
image

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -122,7 +122,7 @@ def create_user_contribution(user:, repository:, date:)
       contrib.company_id = user.company.id
       contrib.repository = repository
       contrib.created_at = date
-      contrib.approve
+      contrib.approve(user.id)
```
Comment (andreLumor) [inline db/seeds.rb:125]: Another option would be leaving seeded contributions at their initial state, but I was concerned about breaking tests.

File: app/admin/contributions.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
   filter :company, as: :select, if: proc { current_user.super_admin? }
   filter :user, as: :select, collection: proc { CompanyUsersCollectionQuery.new(current_user).call }
-  filter :state, as: :select, collection: Contribution.aasm.states_for_select
```
Comment (adbatista) [inline app/admin/contributions.rb:9]: why did you remove this?

File: db/seeds.rb (hunk 1 above)
Comment (adbatista) [reply db/seeds.rb:125]: the seeds file is not used in the tests

File: app/admin/contributions.rb (hunk 2 above)
Comment (andreLumor) [reply app/admin/contributions.rb:9]: Also removed as asked on the issue #130

File: db/seeds.rb (hunk 1 above)
Comment (andreLumor) [reply db/seeds.rb:125]: Oh, ok. I'll remove this to a fix PR

Comment (adbatista) [reply db/seeds.rb:125]: there's no need, is just to let you know that the seed are not loaded in the tests

File: app/admin/contributions.rb [hunk 3]
```diff
@@ -41,7 +41,6 @@
   index download_links: [:xls, :text] do
     selectable_column
     column :user
-    column :company
```
[the remark this answers is not in the document]
Comment (adbatista) [reply app/admin/contributions.rb:44]: instead of removing, change do display this column only is the current_user is superadmin

[the remark this answers is not in the document]
Comment (andreLumor) [reply app/admin/contributions.rb:44]: done!

**Review by adbatista — APPROVED** [review]
